client/my-sites/google-my-business/select-business-type/index.js

```diff
@@ -53,6 +54,10 @@ class SelectBusinessType extends Component {
 
 		return (
 			<Main className="select-business-type">
+				<PageViewTracker
+					path={ '/google-my-business/:site_id/select-business-type' }
```

Comment:
Super minor but we don't need brackets for string attributes,
`<PageViewTracker path="/google-my-business/:site_id/select-business-type" title="Google My Business > Select Business Type" />`

Reply:
We should be using `:site` instead of `:site_id` when tracking pageviews.